iree-prof-tools/iree-vis.cc

```diff
@@ -55,12 +173,10 @@ int main(int argc, char** argv) {
   }
 
   auto* invoke = ireeCompilerInvocationCreate(session);
```

Comment:
You may reference iree-opt-main.cc to see how we can do fine controls.

Reply:
Or, this function: `ireeReduceRunMain()`.

Reply:
Thanks for the reference. Let me address it in a following PR. Added a TODO.